kategory/build.gradle

```diff
@@ -5,14 +5,16 @@ test {
 }
 
 dependencies {
+    compile 'com.github.KindedJ:KindedJ:5bee1acf6f'
     compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlinVersion"
     compile "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlinxCoroutinesVersion"
-    compile "io.kategory:kategory-annotations:$kategoryAnnotationsVersion"
-    kapt "io.kategory:kategory-annotations-compile:$kategoryAnnotationsVersion"
-    kaptTest "io.kategory:kategory-annotations-compile:$kategoryAnnotationsVersion"
-    compileOnly "io.kategory:kategory-annotations-compile:$kategoryAnnotationsVersion"
-    testCompileOnly "io.kategory:kategory-annotations-compile:$kategoryAnnotationsVersion"
+    compile 'com.github.kategory:kategory-annotations:paco-kindedjgen-SNAPSHOT'
```

Comment:
is this correct ? weren't we moving forward to `kategory.io` as our domain? do we have snapshots deployed under a different groupid ?